src/identity-user-interfaces.ts

```diff
@@ -83,15 +83,22 @@ export interface IUserAttributeChangeEvent extends BaseEvent {
 
 // https://go.mparticle.com/work/SQDSDKS-6460
 export interface IIdentityCallback extends IdentityCallback {
-    (result: IdentityResult): void;
+    (result: IIdentityResult): void;
 }
 
 export interface IIdentityResult extends Omit<IdentityResult, 'body'> {
-    body: IIdentityResultBody;
+    body: IIdentityResultBody | IIdentityModifyResultBody;
```

Comment:
If this is omiting the body but then adding a body with a new signature, can't you not omit it and just extend with a new body?

Reply:
Good call. I'll try this out.